mmcls/datasets/pipelines/formating.py

```diff
@@ -122,8 +122,7 @@ def __call__(self, results):
         return data
 
     def __repr__(self):
-        return self.__class__.__name__ + \
-            f'(keys={self.keys}, meta_keys={self.meta_keys})'
+        return self.__class__.__name__ + f'(keys={self.keys})'
```

Comment:
Instead of deleting `meta_keys`, you may consider completing the functionalities regarding `meta_keys`.
Please refer to https://github.com/open-mmlab/mmdetection/blob/fb8ec070f40067c2222f5f15eefdb4d5b1f60bf7/mmdet/datasets/pipelines/formating.py